core/src/main/scala/org/apache/spark/memory/StaticMemoryManager.scala

```diff
@@ -104,6 +104,8 @@ private[spark] class StaticMemoryManager(
 
 private[spark] object StaticMemoryManager {
 
+  private val MIN_MEMORY_BYTES = 450 * 1024 * 1024
```

Comment:
I used the same constant (1.5 * 300 * 1024 * 1024) as the one in the `UnifiedMemoryManager`. 
https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/memory/UnifiedMemoryManager.scala#L190

I'm open to any recommendation here though. :)